bin/varnishtest/tests/b00091.vtc

```diff
@@ -0,0 +1,41 @@
+varnishtest "IPs/port in varnishadm backend.list -j"
+
+feature cmd "jq -V"
+
+server s0 {
+} -start
+
+
+varnish v1 -vcl+backend {
+	backend s1 {
+		.host = "127.0.0.1";
+		.port = "2222";
+		.probe = { .url = "/"; }
+	}
+	backend s2 {
+		.host = "::1:2:3:4";
+		.port = "3333";
+	}
+
+	sub vcl_recv {
+		set req.backend_hint = s0;
+		set req.backend_hint = s1;
+		set req.backend_hint = s2;
+	}
+} -start
+
+shell {
+	set -e
+
+varnishadm -n ${v1_name} backend.list -j
+	varnishadm -n ${v1_name} backend.list -jp > result.json
+	jq '
```

Comment:
why not pipe to jq?

Reply:
I originally voted against piping before this was submitted here for several reasons, mostly because it leaves something to inspect with `varnishtest -l/-L` if the test case ever fails.

Reply:
makes sense, thx